test/per_file_coverage.sh

```diff
@@ -32,7 +32,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/filters/common/fault:94.5"
 "source/extensions/filters/common/rbac:90.8"
 "source/extensions/filters/http/cache:95.9"
-"source/extensions/filters/http/grpc_json_transcoder:94.2" # TODO(#28232)
+"source/extensions/filters/http/grpc_json_transcoder:94.0" # TODO(#28232)
```

Comment:
:-(

Reply:
Not ideal. The PR itself does not change coverage at all, just code reformat alters coverage ratio.